tracee-ebpf/tracee/argprinters.go

```diff
@@ -176,7 +176,68 @@ func (t *Tracee) prepareArgs(ctx *context, args map[string]interface{}) error {
 		if cmd, isInt32 := args["cmd"].(int32); isInt32 {
 			args["cmd"] = helpers.ParseBPFCmd(cmd)
 		}
+	case SecurityKernelReadFileEventID:
+		if readFileId, isUint32 := args["type"].(uint32); isUint32 {
+			typeIdStr, err := ParseKernelReadFileId(int32(readFileId))
+			if err != nil {
+				return err
+			}
+			args["type"] = typeIdStr
+		}
 	}
 
 	return nil
 }
+
+func ParseKernelReadFileId(id int32) (string, error) {
```

Comment:
This function reads the osInfo and initializes kernelReadFileIdStrs accordingly on every call, while osInfo shouldn't change between subsequent calls. Instead, kernelReadFileIdStrs can be initialized once during init.

Reply:
i've added it as part of the Tracee struct. kernelReadFileIdStrs was too specific, so i've added a more generic argParsers field to the Tracee struct, and kernelReadFileIdStrs is part of it. 
now this is initialized when a new Tracee instance is created.

as there is only one argParser currently (kernelReadFileIdStrs), we can check that the event security_kernel_read_file was chosen, before initializing the kernelReadFileIdStrs.

let me if this is what you had in mind.

Reply:
Actually, what I had in mind is using either `sync.Once` or `init()`

Reply:
ohh okay.
i've updated it now to use `init()`.